### ISSUE TITLE
deeplink met akn mogelijk?

### ISSUE THREAD

**Issue by lamerise** [IDgis/PlanoView2-bevindingen#267]
Is een deeplink naar een document in een groep met iets als een akn nummer mogelijk (zoals vroeger met het IMRO-nummer in planoview 1)

**Comment by copierrj** [comment]
Ja, je kunt inderdaad naar een specifiek document linken door het akn id in de url op te nemen:

https://omgevingsdocumenten.zuid-holland.nl/omgevingsbeleid?d=/akn/nl/act/pv28/2024/omgevingsvisie-1/nld@2024-04-25;4

Ook is het mogelijk om daarbij ids van IMOW objecten mee te geven die dan direct op de kaart zichtbaar worden:

https://omgevingsdocumenten.zuid-holland.nl/omgevingsbeleid?d=/akn/nl/act/pv28/2024/omgevingsvisie-1/nld@2024-04-25;4&o=nl.imow-pv28.gebiedengroep.5d5b2a1bfc3f4d85bfd0e151c18ae0a7
